Makefile

```diff
@@ -0,0 +1,19 @@
+test:
```

Comment:
It would be nice to be able to specify a `file` argument to run only specific tests.
That's how I would do it:
https://github.com/Tastyep/structlog.nvim/blob/main/Makefile#L2
https://github.com/Tastyep/structlog.nvim/blob/main/script/test.sh

Reply:
Yeah for sure, I can just add additional targets.